Purchases/Purchasing/TrialOrIntroPriceEligibilityChecker.swift

```diff
@@ -109,6 +92,28 @@ class TrialOrIntroPriceEligibilityChecker {
             }
     }
 
+    // swiftlint:disable line_length
+    fileprivate func fetchIntroEligibilityFromBackend(with receiptData: Data,
+                                                      productIdentifiers: [String],
+                                                      completionBlock receiveEligibility: @escaping ReceiveIntroEligibilityBlock) {
```

Comment:
tbh I'd just kill `receiveEligibility` as a label and use `completion` instead, since it doesn't add much clarity, and that also gets rid of the `line_length` problem
Also, we usually separate the `private` stuff into a private extension

Reply:
I am going to kill it too in the public method, it will be another breaking change but I think it's good for consistency

Reply:
the public method already had an update I believe

Reply:
https://github.com/RevenueCat/purchases-ios/blob/main/Purchases/Public/Purchases.swift#L944

Reply:
Actually, it's already `completion` in `main` lol